releng/org.eclipse.nebula.examples.release.feature/feature.xml

```diff
@@ -412,5 +412,11 @@
          install-size="0"
          version="0.0.0"
          unpack="false"/>	
-		 
+

```

Comment:
Laurent, snippets should not go in the example features unless they contain contributions to the Nebula Examples view. I believe all these plugins are just snippets.